support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/input/ApproverAction/summary.jelly

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License
+
+Copyright (c) 2013-2014, CloudBees, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+ <j:jelly xmlns:j="jelly:core" xmlns:t="/lib/hudson">
+   <t:summary icon="orange-square.png">
+        ${%approved_by_user(it.userId, it.userName, rootURL)}.
```

Comment:
The hyperlink is not going to work when you have `escape-by-default='true'`; `<` will become `&lt;`. You can solve that using `<j:out value="${…}"/>` but then you *must be careful* to call `h.escape(…)` on the user’s display name, so that they cannot launch a (stored) XSS attack.

It would be wise to use HtmlUnit (`JenkinsRule.WebClient`) to verify that the expected approval text, including valid hyperlink, in fact appears on the build index page.